### ISSUE TITLE
Unable to resolve remapped inputs in a Truffle/Foundry project

### ISSUE THREAD

**Issue by jameswenzel** [NomicFoundation/hardhat-vscode#315]
`foundry.toml`

```toml
[profile.default]
src = 'contracts'
out = 'out'
libs = ['lib', 'node_modules']
```

`remappings.txt.`

```
@manifoldxyz/=node_modules/@manifoldxyz/
@openzeppelin/=node_modules/@openzeppelin/
ds-test/=lib/forge-std/lib/ds-test/src/
forge-std/=lib/forge-std/src/
```
<img width="622" alt="Screenshot 2022-12-16 at 11 49 34 AM" src="https://user-images.githubusercontent.com/6371847/208177699-36be96b6-59c7-4f51-b347-470f0c44043b.png">

This is on the `main` branch of [royalty-registry-solidity](https://github.com/manifoldxyz/royalty-registry-solidity) after running `forge init --force`, modifying it to point at `src = 'contracts'` and `libs = ['lib', 'node_modules']`, and adding a `remappings.txt`.

**Comment by kanej** [comment]
@jameswenzel hey, I wasn't able to find a foundry/remappings.txt on the main branch of https://github.com/manifoldxyz/royalty-registry-solidity

Are these changes that haven't been pushed?

**Comment by jameswenzel** [comment]
Yes, I added the `remappings.txt` myself using `forge remappings > remappings.txt`

However, it seems like this has been resolved in the meantime - possibly due to #308, which was merged around the time I opened this issue. 

I'll reopen if I encounter again, thanks!
